source/getting-started/control-with-amdc/encoder-fb/index.md

```diff
@@ -1,13 +1,97 @@
+
+Example code to convert encoder to angular position in radians:
+```C
+double task_get_theta_m(void)
+{
+    // Get raw encoder position
+    uint32_t position;
+    encoder_get_position(&position);
+
+    int ENCODER_COUNT_PER_REV, CCW;
+    double enc_theta_m_offset;
+
+    // User to set encoder count per revolution
+    ENCODER_COUNT_PER_REV = 1024;
+
+    // Set 1 if encoder count increases with CCW rotation of shaft, Set 0 if encoder count increases with CW rotation of shaft
+    CCW = 1; 
+
+    // Angular position to be computed
+    double theta_m_enc;
+
+    // User to set encoder offset
+    enc_theta_m_offset = 100;
+
+
+    // Convert to radians
+    if (CCW){
+        theta_m_enc = (double) PI2 * ( ( (double)position - enc_theta_m_offset )/ (double) ENCODER_COUNT_PER_REV);
+    }
+    else{
+        theta_m_enc = (double) PI2 * ( ( (double)ENCODER_COUNT_PER_REV - (double)1 -(double)position + enc_theta_m_offset )/ (double) ENCODER_COUNT_PER_REV);
+    }
+    
+     // Mod by 2 pi
+    theta_m_enc = fmod(theta_m_enc,PI2 );
+    return theta_m_enc;
+}
+```
+
+
+
+### Finding the offset
+
+The example code shown above makes uses of an encoder offset value. It is necessary for the user to find this offset experimentally for their machine. For synchronous machines, this offset is the count value measured by the encoder when the d-axis of the rotor is aligned with the phase U winding axis of the stator. 
+
+To get the rotor to align with the phase U winding, the user may have some current flow through phase U and out of phase V and phase W. This can done by using a DC supply, with phase U connected to the positive terminal and phases V and W connected to the negative terminal. Alternately, the user may also inject some current along the d-axis using the AMDC [Signal Injection](https://docs.amdc.dev/getting-started/user-guide/injection/index.html) module. While injecting d-axis current, the user must ensure that the rotor position is set to zero in the control code. After obtaining the offset, the user needs to set the variable `enc_theta_m_offset` to the appropriate value in the `task_get_theta_m()` function. 
```

Comment:
Can you make this a bit easier to read? This somehow reads complicated when it is actually quite a simple procedure. Say something like: 

> To determine the appropriate offset value (reported as `enc_theta_m_offset` in the code example above), eliminate any source of load torque on the shaft and apply a large current vector at 0 degrees ($I_u = I_0$, $I_v = I_w = -\frac{1}{2} I_0$). This should cause the rotor to align with the phase u axis. The offset value can now be obtained as `enc_theta_m_offset = encoder_get_position()`.

(Do I have this right? Please edit / fix as appropriate)